lib/sys.py

```diff
@@ -24,6 +24,8 @@
 
 goversion = Version()
 maxint = MaxInt
+maxsize = maxint
+maxunicode = 0xffff
```

Comment:
I guess re implementation won't work if it is 32bit I guess. may be hard code in re, instead of here?